<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnkidownImportDialog</class>
 <widget class="QDialog" name="AnkidownImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ankidown - Import</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="inputTab">
        <attribute name="title">
         <string>Input</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="chooseFile">
               <property name="text">
                <string>Choose File</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="fileLabel">
               <property name="text">
                <string>Choose a file...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="noteTextEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="previewButton">
             <property name="text">
              <string>Preview</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="templateTab">
        <attribute name="title">
         <string>Template</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPlainTextEdit" name="templateTextEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QWidget" name="modelArea" native="true"/>
         </item>
         <item>
          <widget class="QWidget" name="deckArea" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="fieldsArea" native="true"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
